docs/pages/choose-an-edition/teleport-cloud/getting-started.mdx

```diff
@@ -58,7 +58,7 @@ Your terminal will begin a shell session in the `ubuntu:22.10` container. From
 there, make sure you have installed `curl`:
 
 ```code
-$ apt update && apt install -y curl
+$ apt update && apt install -y curl telnet
```

Comment:
I'm not seeing the value here. Why are we asking the user to install a tool that we never mention again in the rest of the guide?

Reply:
I think the idea is that the automatic join script can use it, but only if it's installed: https://github.com/gravitational/teleport/blob/master/lib/web/scripts/node-join/install.sh#L267-L269

Reply:
I added a description on its usage